packages/test/test-end-to-end-tests/src/test/container.spec.ts

```diff
@@ -137,6 +145,18 @@ describeNoCompat("Container", (getTestObjectProvider) => {
 			"Client details should be set with interactive as true",
 		);
 	});
+	it.only("Delta manager receives pong event", async () => {
+		if (provider.driver.type !== "local" && provider.driver.type !== "odsp") {
```

Comment:
Can you please add comment on why this check exists? 
FRS should be good too, right?
I wonder how it works for local driver type - it should not use same driver base. Does it actually work?